test/UnitTests/TestFramework.UnitTests/Assertions/AssertTests.AreEqualTests.cs

```diff
@@ -1140,4 +1142,217 @@ public bool Equals(A? other)
         public override int GetHashCode()
             => Id.GetHashCode() + 1234;
     }
+
+    public void AreEqualStringDifferenceAtBeginning()
```

Comment:
if we have a multiline string, should we show line markers?

Reply:
Multiline string will currently escape to show the newlines as printable characters using the control pictures class of characters (specifically ␍ and  ␊).

I would prefer to have some mode where comparing multiline strings is easier, but the best way to do that is to have a general diffing mechanism and show all differences in the text (imho).

How to effectively do that in console idk, especially when we cannot really depend on colors etc.